icu4j/pom.xml

```diff
@@ -121,9 +121,8 @@
       </roles>
     </developer>
     <developer>
-      <id>yumaoka</id>
-      <name>Yoshito Umaoka</name>
-      <organization>IBM</organization>
+      <id>richard_gillam</id>
```

Comment:
@richgillam -- Mihai picked this from your email address (from `git log`). Apparently this is kind of arbitrary. Would you prefer "richgillam" for your GitHub ID?

Reply:
The spec: https://maven.apache.org/pom.html#developers
> "id, name, email: These correspond to the developer's ID (presumably some unique ID across an organization), the developer's name and email address."

I tried to keep the info minimal (and consistent with what we had before).
The `id` is mandatory, the email not.

Reply:
> Would you prefer "richgillam" for your GitHub ID?

Well, my GitHub ID *is* "richgillam".  Or is that your point: Use "richgillam" from the GitHub ID as opposed to "richard_gillam" from my (Apple) email address?  I think I like "richgillam" because it's shorter, but I'm not sure it matters that much.

Reply:
> > Would you prefer "richgillam" for your GitHub ID?
> 
> Well, my GitHub ID _is_ "richgillam". Or is that your point: Use "richgillam" from the GitHub ID as opposed to "richard_gillam" from my (Apple) email address?

Well, I _tried to make that point_ but wasn't clear.

> I think I like "richgillam" because it's shorter, but I'm not sure it matters that much.

I suggest that we use GitHub IDs here and add a comment line about it.

Reply:
Done